circuit_passes/src/bucket_interpreter/env/extracted_func_env.rs

```diff
@@ -159,7 +206,11 @@ impl<'a> ExtractedFuncEnvData<'a> {
                             }) => value,
                             _ => unreachable!(), //ASSERT: 'cmp_address' was formed by 'loop_unroll::new_u32_value'
                         };
-                        self.base.subcmp_counter_is_zero(subcmp)
+                        if counter_override {
+                            todo!()
```

Comment:
Weird, when I reorganized commits I guess it pulled this change in twice because you pointed it out in the other PR too. I'll put the change here as well so it should merge automatically.